app/scripts/modules/ecs/src/serverGroup/configure/wizard/taskDefinition/taskDefinition.html

```diff
@@ -24,12 +24,17 @@
                 type="radio"
                 ng-model="command.useTaskDefinitionArtifact"
                 ng-value="true"
+                ng-disabled="command.viewState.mode == 'create'"
                 id="taskDefinitionSourceArtifact"
               />
               Artifact
             </label>
           </div>
         </div>
+        <div ng-if="command.viewState.mode == 'create'" style="color:#666;">
```

Comment:
```suggestion
        <div ng-if="command.viewState.mode === 'create'" class="color-text-caption">
```
The `===` is super minor; the color selection via CSS class is based off the style guide (accessible when running locally via http://localhost:9000/#/styleguide#psg-link-0).

Reply:
Makes perfect sense! Will update.